### PR TITLE
refactor: removing huh as a dep

### LINKED ISSUE #741: refactor: remove huh as a dependency
`gum choose`
- add header
- add filter
combine `gum choose` and `gum filter`
- full screen option
- select/unselect all

bash to check what bugs have been fixed in prev versions of huh and gum to test against this new branch

### PR DESCRIPTION
**Pull request by caarlos0**
fixes #741

### CODE REVIEW

**Review by aymanbagabas — APPROVED** [review]
Yass!! 🙌

**Review by meowgorithm — APPROVED** [review]
_BEAST_!

File: confirm/confirm.go [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+	Help        key.Binding
+}
+
+// FullHelp implements help.KeyMap.
+func (k keymap) FullHelp() [][]key.Binding {
+	return [][]key.Binding{
+		{k.Toggle, k.Submit, k.Affirmative, k.Negative},
+		{k.Abort, k.Quit, k.Help},
+	}
+}
```
Comment (caarlos0) [inline confirm/confirm.go:82]: i wonder if need full help? didn't add to the other components, I think we either add it in all or none of them, ideally?

wdyt @meowgorithm

Comment (meowgorithm) [reply confirm/confirm.go:82]: Hmm. I think you're right that it's generally extraneous, though in some cases I think it's helpful. Why don't we hide it by default across the board and eventually aim to get it into all the components that make sense?

**Pull request merged**